config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow_devmode.yaml

```
@@ -6,6 +6,8 @@ metadata:
     sw.kogito.kie.org/description: Greeting example on k8s!
     sw.kogito.kie.org/version: 0.0.1
     sw.kogito.kie.org/profile: dev
+  labels:
+    ext_camel: mycamel-configmap
```

```suggestion
    sw.kogito.kie.org/resource/camel: mycamel-configmap
```

We must use the correct namespace here.

---

ok

---

Yes, we can't have multiple paths. My bad. I think `sw.kogito.kie.org/resource-camel` should do. :)